src/main/webapp/app/orion/participation/orion-course-exercise-details.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'jhi-orion-course-exercise-details',
+    templateUrl: './orion-course-exercise-details.component.html',
+})
+export class OrionCourseExerciseDetailsComponent {
+    // only replaces the student actions with Orion student actions in the overview
+}
```

Comment:
[nit] What do you think of inlining the html, as it is small and the component itself has no typescript code

Reply:
Makes sense, inlined the html :+1: